src/views/skuLookup.tsx

```diff
@@ -0,0 +1,349 @@
+              />
+              <SkuCard
+                title={SKU_INVENTORY.ATG_INVENTORY}
+                cardData={atgInventory}
+              />
+              <SkuCard
+                title={SKU_INVENTORY.YANTRIKS_INVENTORY}
+                cardData={yantriksInventory}
+              />
+              <SkuCard title={SKU_PRODUCT_INFO.TITLE} cardData={productInfo} />
+            </div>
+          ) : (
+            <>
+              <HorizontalTable
+                tableData={skuDetailsTable}
+                tableHead={SKU_DETAILS.TITLE}
+                rowHeadClassName="p-4 text-right font-medium w-[15%] lg:whitespace-nowrap"
+                dataClassName="text-left p-4 border-r border-black-400 min-w-[15%]"
+              />
+              <HorizontalTable
+                tableData={priceDetailsTable}
+                tableHead={SKU_PRICE_DETAILS.TITLE}
+                rowHeadClassName="p-4 text-right font-medium w-[12.5%] lg:whitespace-nowrap"
+                dataClassName="text-left p-4 border-r border-black-400 w-[12.5%]"
+              />
+              <HorizontalTable
+                tableData={atgInventory}
+                tableHead={SKU_INVENTORY.ATG_INVENTORY}
+                rowHeadClassName="p-4 text-right font-medium w-[15%] lg:whitespace-nowrap"
+                dataClassName="text-left p-4 border-r border-black-400 w-[15%]"
+              />
+              <HorizontalTable
+                tableData={yantriksInventory}
+                tableHead={SKU_INVENTORY.YANTRIKS_INVENTORY}
+                rowHeadClassName="p-4 text-right font-medium w-1/4 lg:whitespace-nowrap"
+                dataClassName="text-left p-4 border-r border-black-400 w-1/4"
+              />
+              <HorizontalTable
+                tableData={productInfo}
+                tableHead={SKU_PRODUCT_INFO.TITLE}
+                rowHeadClassName="p-4 text-right font-medium w-[15%] lg:whitespace-nowrap"
+                dataClassName="text-left p-4 border-r border-black-400 w-[15%]"
+              />
+            </>
+          )}
+        </>
+      )}
+    </div>
+  );
+};
+
+const SkuCard = (props: SkuCardProps) => {
+  return (
+    <>
+      <Header title={props.title} />
+      {props.cardData?.length > 0 &&
+        props.cardData?.map((obj, index) => {
+          return (
+            <CustomCard
+              key={index}
+              className="sku-lookup-card my-1 text-gray-200 bg-black-200 w-full rounded-lg"
+            >
+              <div
+                className={`grid grid-cols-[repeat(auto-fill,minmax(91px,1fr))] gap-2 ${props.wrapperClass}`}
+              >
+                {Object.entries(obj)?.map(
+                  (
+                    [key, value]: [
+                      string,
+                      Array<string | number> | string | number,
+                    ],
+                    i
+                  ) => (
+                    <div key={i}>
```

Comment:
`<Fragment key={i}>`

Reply:
div is needed as a wrapper to both the other div's